data/seed.yaml

```diff
@@ -71,9 +71,6 @@ spec:
           # Datacenter location, e.g. "ams3". A list of existing datacenters can be found
           # at https://www.digitalocean.com/docs/platform/availability-matrix/
           region: ""
-        # EnforceAuditLogging enforces audit logging on every cluster within the DC,
-        # ignoring cluster-specific settings.
-        enforceAuditLogging: false
```

Comment:
Why did you change this?

Reply:
I was trying to make it more readable, didn't know it's auto generated The keys are sorted alphabetically, so some of the spec flags are listed between the cloud provider keys. will revert this.